### PR TITLE
Maintain the date range when jumping between spaces

### PR DESCRIPTION
**Pull request by beweinreich**
Let me know if I need to clean up the `explore-space-trends.ts` page logic. But, essentially, this means that once you select a date range, you maintain that date range.

Apparently, this was driving customers nuts, since the date range would reset every time they clicked on a space.

### CODE REVIEW

File: src/actions/route-transition/explore-space-trends.ts [hunk 1]
```diff
@@ -82,13 +82,23 @@ export default function routeTransitionExploreSpaceTrends(id) {
     dispatch(collectionSpacesSetDefaultTimeRange(selectedSpace));
     dispatch(collectionSpacesSet(spaces));
 
+    let startDate, endDate;
+    let state = getState()
+    if(state.spaces.filters.startDate != null && state.spaces.filters.endDate != null) {
```
Comment (1egoman) [inline src/actions/route-transition/explore-space-trends.ts]: Can you make this `if (`? Sorry, one of those things that drives me crazy 😄

File: src/reducers/spaces/index.ts [hunk 2]
```diff
@@ -205,26 +205,30 @@ export default function spaces(state=initialState, action) {
...
-        // data
-        startDate: formatInISOTime(
-          getCurrentLocalTimeAtSpace(action.space).subtract(1, 'week').startOf('week')
-        ),
-        endDate: formatInISOTime(
-          getCurrentLocalTimeAtSpace(action.space).subtract(1, 'week').endOf('week')
-        ),
-      },
-    };
+    if(state.filters.date != null && state.filters.startDate != null && state.filters.endDate != null) {
```
Comment (1egoman) [inline src/reducers/spaces/index.ts]: here too

File: src/actions/route-transition/explore-space-trends.ts (hunk 1 above)
Comment (beweinreich) [reply src/actions/route-transition/explore-space-trends.ts]: yep np problem

File: src/reducers/spaces/index.ts (hunk 2 above)
Comment (beweinreich) [reply src/reducers/spaces/index.ts]: got it

**Pull request merged**